getting-started/vs-integration/new-project-wizard.md

```diff
@@ -12,6 +12,8 @@ This article demonstrates how to use the Telerik Visual Studio extensions to cre
 
 To create a new Telerik UI for Blazor application, use the Create New Project wizard. The wizard detects all installed versions of Telerik UI for Blazor and lists them in the **Version** combobox&mdash;this feature enables you to start your project with the desired version. To make sure the components are up to date, you can also [get the latest version]({%slug getting-started-vs-integration-latest-version%}).
 
+## How It Works
+
 The Create New Project Wizard automates a set of steps that you otherwise must perform manually.
 
 The wizard adds the following client assets to your project:
```

Comment:
I recommend using ending punctuation consistently across the two lists (that is all items in both lists end with a period). Otherwise, when rendered they seem a bit off.